lerna.json

```diff
@@ -3,7 +3,10 @@
   "changelogPreset": "conventional-changelog-conventionalcommits",
   "command": {
     "publish": {
-      "message": "chore: publish"
+      "message": "chore: publish",
+      "fromPackage": true,
```

Comment:
does this mean e.g. for a `lavamoat-core` core update

since 4 other pkgs depend on it
- ref: https://github.com/LavaMoat/LavaMoat/blob/main/RELEASE.md#release-dependency
- prev bumps: https://github.com/LavaMoat/LavaMoat/pull/610

we'll create 4 new `separate-pull-requests` to update them?
or these will be included in the original PR?

ref: https://lerna.js.org/docs/features/version-and-publish#from-package

Reply:
@leotm I'm not sure I understand the context of asking this in the lerna config

release-please is configured to create separate pull requests for each public workspace (package).  we are in control of how or when those get merged, and are also in control of how or when those get finally published.  but they all must be published at once!

so when we merge a change to `lavamoat-core`, a PR is created for that.  once _that_ PR is merged, then PRs will be created for each of the packages which depend on the new version of `lavamoat-core`.  **it would be a mistake to merge and publish _those_ before publishing the new version of `lavamoat-core`** since they will not exist in the registry.  because lerna is configured to publish _everything that needs publishing_, we avoid this problem. so publishing in any other way besides this is just asking for trouble.

This may be why the default behavior of release-please is to _not_ create separate pull requests, because it may become confusing.  if it turns out we don't like separate PRs, I propose we simply...disable this behavior. 😄 

cc @naugtur @legobeat @weizman just so you see this